src/features/commentsSlice.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-param-reassign */
+
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import * as commentsApi from '../api/comments';
+import { Comment } from '../types/Comment';
+
+export interface CommentState {
+  items: Comment[],
+  loaded: boolean,
```

Comment:
```suggestion
  comments: Comment[],
  isLoaded: boolean,
```
- You can set corresponding name in the state instead of renaming in components
- Boolean variables/properties names should start from `is` or `has`